lib/ansible/module_utils/storage/hpe3par/hpe3par.py

```diff
@@ -0,0 +1,90 @@
+# Copyright: (c) 2018, Hewlett Packard Enterprise Development LP
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from ansible.module_utils import basic
+
+
+def convert_to_binary_multiple(size_with_unit):
```

Comment:
Would `human_to_bytes` from `module_utils/basic.py` be of use instead?

Reply:
We have changed the method to use `human_to_bytes` internally in this method.

`size_kib = basic.human_to_bytes(size)`

The only addition here is that we are changing the user provided units (MiB, KiB, GiB) into M, T and K that human_to_bytes method expects.